### PR TITLE
Update creating steps doc

### PR DESCRIPTION
**Pull request by hatchan**

### CODE REVIEW

**Comment by mies** [comment]
looks good. ready to rock

**Comment by Tonnu** [comment]
+1 :100:

File: content/docs/steps/creating-steps.md [hunk 1]
```diff
@@ -138,12 +99,15 @@ this such that people can easily deduct what your step does and how to use it.
 
 ### Publishing your step
 
+> Note: currently it is only possible to publish steps using the old
+infrastructure. Published steps will work on both stacks though.
```
Comment (lindseybateman) [inline content/docs/steps/creating-steps.md]: link the word stacks to the stacks article on docs

Comment (hatchan) [inline content/docs/steps/creating-steps.md]: Added a link

**Pull request merged**